examples/complete/locals.tf

```diff
@@ -0,0 +1,34 @@
+locals {
+  name = "example-complete-role"
+  tags = {
+    environment        = "examples"
+    "user::CostCenter" = "terraform-registry"
+  }
+  role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
```

Comment:
Policies `Sid` is missing, we should use it to identify the purpose/use case of any policy